go.mod

```
@@ -1,9 +1,8 @@
 module github.com/minio/hperf
 
-go 1.19
+go 1.22.5
```

```suggestion
go 1.20
```
.. or are we using newer features? No need to bump this to latest.

---

This is one of those go mod tidy bugs.

---

There are vuln. in older versions and the vuln checker was failing. That's the only reason really.

---

ah then stick it to go 1.21

---

1.21 also had vulns